testgrid/cmd/entomologist/main.go

```diff
@@ -78,15 +78,29 @@ func (o *options) validate() error {
 	if o.repository == "" {
 		return errors.New("--github-repo is required")
 	}
-	if o.gcsPath == "" || !strings.HasPrefix(o.gcsPath, "gs://") {
-		return errors.New("invalid or missing --output")
+	if outputErr := validateOutput(o.output); outputErr != nil {
+		return outputErr
 	}
-	if o.gcsCredentials == "" {
+	if strings.HasPrefix(o.output, "gs://") && o.gcsCredentials == "" {
 		return errors.New("--gcs-credentials-file required for write operations")
 	}
 	return o.github.Validate(false)
 }
 
+func validateOutput(output string) error {
+	if output == "" {
+		return errors.New("missing --output")
+	}
+	if _, err := os.Stat(output); err != nil && !strings.HasPrefix(output, "gs://") {
```

Comment:
Checking if a file can be opened as an output isn't appropriate for two reasons:
1) It's likely that a user would want to create a new file and output there (`--output=some_new_file.txt`), and this code would return a "does not exist" error. [Example](https://stackoverflow.com/questions/12518876/how-to-check-if-a-file-exists-in-go)
2) The file might change state between option validation and actually writing to it

Simply validating that the command _looks_ like a valid GCS or local path is sufficient here